src/dashboard/src/settings/base.py

```diff
@@ -437,8 +443,13 @@
 
 # Dashboard internal settings
 GEARMAN_SERVER = config.get("gearman_server")
-POLLING_INTERVAL = 5  # Seconds
-STATUS_POLLING_INTERVAL = 5  # Seconds
+
+DEFAULT_POLLING_INTERVAL = 10
```

Comment:
Hey Ross, config defaults are defined in `CONFIG_DEFAULTS` above.

Reply:
sorry @sevein have you an example how to use that here?

Reply:
I.e. `config.get("polling_interval")` is going to return `10` when the user doesn't provide a value since you indicated it in `CONFIG_DEFAULTS`.

Reply:
Right, it does - I was testing that. But Tessa raised the issue of casting in the PR commentary below, and if config.get("polling_interval") can't cast either of those to an integer then the default won't work like that. So I suspect I am missing something - but also, have a look at my comment around the try/except here. I have only added this "default" for that particular last commit.

Reply:
`appconfig.Config` is using [getint](https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.getint) for coercion under the covers for `int` attributes like your new `polling_interval` so we don't have to worry.

Reply:
I don't think it's working how you think it is so that's why I'd ask you to try it. 

If I set `ARCHIVEMATICA_DASHBOARD_DASHBOARD_POLLING_INTERVAL: "fff"`: I get this error:

```
archivematica-dashboard_1        |   File "/src/archivematicaCommon/lib/env_configparser.py", line 14, in wrapper
archivematica-dashboard_1        |     return fn(*args, **kwargs)
archivematica-dashboard_1        |   File "/src/archivematicaCommon/lib/env_configparser.py", line 70, in getint
archivematica-dashboard_1        |     return ConfigParser.SafeConfigParser.getint(self, *args, **kwargs)
archivematica-dashboard_1        |   File "/usr/local/lib/python2.7/ConfigParser.py", line 359, in getint
archivematica-dashboard_1        |     return self._get(section, int, option)
archivematica-dashboard_1        |   File "/usr/local/lib/python2.7/ConfigParser.py", line 356, in _get
archivematica-dashboard_1        |     return conv(self.get(section, option))
archivematica-dashboard_1        | ValueError: invalid literal for int() with base 10: 'fff'
```
Really just trying to honor the code-review here and I've left a note about what that mean, or not mean here. Have a think about the intention and let me know.

Reply:
Ok, I see. I appreciate the effort. I read Tessa's suggestion but misssed [your comment](https://github.com/artefactual/archivematica/pull/1643#discussion_r520666589). I think that the level of strictness is beneficial, this is the only window we have to report errors that is not disruptive. Fail-fast as opposed to masking the issue, if that makes sense. It'd be nice though if the env_configparser reported a friendlier error that includes the section and option values from the config parser.